xdsl/dialects/llvm.py

```diff
@@ -366,6 +410,96 @@ class LLVMMLIRUndef(IRDLOperation):
     res: Annotated[OpResult, AnyAttr()]
 
 
+@irdl_op_definition
+class GlobalOp(IRDLOperation):
+    name = "llvm.mlir.global"
+
+    global_type: OpAttr[Attribute]
+    constant: OptOpAttr[UnitAttr]
+    sym_name: OpAttr[StringAttr]
+    linkage: OpAttr[LinkageAttr]
+    dso_local: OptOpAttr[UnitAttr]
+    thread_local_: OptOpAttr[UnitAttr]
+    value: OptOpAttr[Attribute]
+    alignment: OptOpAttr[AnyIntegerAttr]
+    addr_space: OpAttr[AnyIntegerAttr]
+    unnamed_addr: OptOpAttr[AnyIntegerAttr]
+    section: OptOpAttr[StringAttr]
+
+    # This always needs an empty region as it is in the top level module definition
+    body: Region
+
+    @staticmethod
+    def get(global_type: Attribute,
```

Comment:
nit: make init instead

Reply:
I hope you don't mind, I am going to keep with get for the moment as this will be consistent with all other operations in the dialect. Later on we can move over to using init throughout the dialect